src/scala/org/pantsbuild/scoverage/report/ScoverageReport.scala

```diff
@@ -0,0 +1,252 @@
+package org.pantsbuild.scoverage.report
+
+import java.io.File
+import org.pantsbuild.zinc.options.Parsed
```

Comment:
Would this necessarily belong to pants repo? Can this be published as a JAR and code potentially living in scoverage or temporarily in com.twitter.scoverage instead. If so, could remove dependencies on pantsbuild and use a more scoverage OSS-friendly options parser library.